Assets/Scripts/Models/Furniture.cs

```diff
@@ -58,6 +58,8 @@ public class Furniture : IXmlSerializable, ISelectable, IContextActionProvider
 
     private List<Job> jobs;
 
+    private List<Job> jobsQueued;
```

Comment:
We should call these pausedJobs, or something different. Since now it seems the same as the jobs list.